src/docs/product/data-management-settings/server-side-sampling/index.mdx

```diff
@@ -0,0 +1,109 @@
+
+![Visualization of how Sentry samples from total transaction volume down to post server sampling volume](how-sentry-samples.png)
+
+In this analogy, the bucket of rocks is sifted through two sieves. Both sieves can hold the total number of rocks, but the first sieve allows more rocks to pass through because it's coarser than the second. The first sieve symbolizes the process of sampling with the SDK, which, in our example, samples half of the total number of transactions the SDK observes. Symbolizing the last sieve, server-side sampling receives the half of the total transactions and then filters for a quarter of the total transactions observed.
+
+<!-- prettier-ignore-start -->
+
+| Sampling Process | Rate | Count |
+|------------------|------|-------|
+|Total transactions observed | N/A | 1000 |
+| Sentry SDK sampled | 50% | 500 |
+| Sentry servers sampled | 25% | 250 |
+
+<!-- prettier-ignore-end -->
+
+Now in reality, transactions don't vary in size like a rock, but the sieve analogy illustrates how many accepted transactions are in the end calculated.
+
+Below, you can see how an incorrect configuration of your SDK and server-side sampling rates would affect the transactions received. The sieves are reversed, and the SDK allows for a quarter of the total number of transactions to pass through. In this analogy, the coarser sieve now assigned for server-side sampling, wouldn't filter any more transactions because it's coarser than its counterpart. To summarize, the sampling rate configured for your SDK needs to be equal or greater than the rate set on Sentry servers.
+
+<!-- prettier-ignore-start -->
+
+| Sampling Process | Rate | Count |
+|------------------|------|-------|
+|Total transactions observed | N/A | 1000 |
+| Sentry SDK sampled | 25% | 250 |
+| Sentry servers sampled | 50% | 250 |
+
+<!-- prettier-ignore-end -->
+
+## Traces & Propagation of Sampling Decisions
+
+TBA
+
+## Server-Side Sampling Configurations
+
+<!-- TO DO: Decide if this should be a child page or remain on this page -->
+
+Now that you know how Sentry is sampling at a high-level, you can set server-side sampling rules in **[Project] > Settings > Server-side Sampling**.
+
+### Order of Precedence
+
+Rules evaluate from top to bottom, with the first matching rule determining the sample rate. Like with declaring a conditional statement in code, rules with the most specific conditional clauses should be evaluated first (`If`), followed by more generic ones (`Else If`). If generic rules lead the list, the more specific rules will likely never be evaluated. Setting a uniform sampling rate is equivalent to an `Else` clause which will always trail rules with conditional clauses.
+
+The Server-side Sampling page in sentry.io allows you to change the order of your rules so you can set it effectively.
+
+### Good Examples
```

Comment:
I like this in general, but shouldn't we point out that this is just pseudo code as the rules are set via UI?
I may get confused wondering where to put that.